README.md

```diff
@@ -63,8 +63,90 @@ Check the output
 
 * Locate the output on disk at `~/aircan-example-1.json`
 
+## Using Aircan DAGs
 
-### Example 2: Local file to CKAN DataStore
+
+### Example 2: Local file to CKAN DataStore using the Datastore API
+
+#### Preliminaries: Setup your CKAN instance
+
+We'll assume you have:
+
+* a local CKAN setup and running at https://localhost:5000
+* a dataset (for example, `my-dataset`) with a resource (for example, `my-resource` with the `my-res-id-123` as its `resource_id`).
+* We also need to set up two environment variables for Airflow. Access the Airflow Variable panel and set up `CKAN_SITE_URL` and your `CKAN_SYSADMIN_API_KEY`:
+
+![Variables configuration](docs/resources/images/aircan_variables.png)
+
+
+Now you can run the `ckan_api_load_single_step` by following these steps:
+
+1. Open your `airflow.cfg` file (usually located at `~/airflow/airflow.cfg` and point your DAG folder to aircan:
+```bash
+dags_folder = /your/path/to/aircan/dags
+```
+
+2. Save the file and verify that Airflow finds the DAG by running `airflow list_dags`. The output should list 
+```bash
+-------------------------------------------------------------------
+DAGS
+-------------------------------------------------------------------
+ckan_api_load_single_step
+...other DAGs...
+```
+
+3. Make sure you have these environment variables properly set up:
+```bash
+export LC_ALL=en_US.UTF-8
+export LANG=en_US.UTF-8
+export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES
+```
+
+4. Run the airflow webserver: `airflow webserver`
```

Comment:
This should have been happening as part of airflow setup (?) Or do we have to redo because we reset the dags folder?